cmd/login/flag.go

```diff
@@ -48,5 +50,14 @@ func (f *flag) Init(cmd *cobra.Command) {
 }
 
 func (f *flag) Validate() error {
+	// Validate ttl flag
+	ttlFlag, err := time.ParseDuration(f.WCCertTTL)
+	if err != nil {
+		return microerror.Maskf(invalidFlagError, `--%s is not a valid duration. Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".`, flagWCCertTTL)
```

Comment:
Just a passerby comment. ns, us, µs, ms are all not really relevant units for our use case.